src/components/views/read/cvForm/Education.tsx

```diff
@@ -126,206 +155,165 @@ const Education: FC<Props> = ({
-                                }}
-                              />
-                            </LocalizationProvider>
-                          )}
-                          {m.key === 'endDate' && (
-                            <LocalizationProvider dateAdapter={AdapterDayjs}>
-                              <DatePicker
-                                label="Till"
-                                value={
-                                  // endDate
-                                  inputValues.find(
-                                    (inputValue) =>
-                                      inputValue.cardId === card.id,
-                                  )?.values.endDate || endDate
-                                }
-                                minDate={startDate}
-                                maxDate={dayjs()}
-                                onChange={(date) => {
-                                  handleChange(
-                                    card.id,
-                                    'endDate',
-                                    date
-                                      ? dayjs(date).format('YYYY-MM-DD')
-                                      : dayjs(endDate).format('YYYY-MM-DD'),
-                                  );
-                                  setEndDate(dayjs(date));
-                                }}
-                              />
-                            </LocalizationProvider>
-                          )}
-                          {m.key === 'country' && (
-                            <TextField
-                              id="select-country"
-                              select
-                              label="Country"
+                      )}
+                      {m.key === 'startDate' && (
+                        <LocalizationProvider dateAdapter={AdapterDayjs}>
+                          <DatePicker
+                            label="From"
+                            value={
+                              card.startDate ? dayjs(card.startDate) : undefined
+                            }
+                            maxDate={dayjs()}
+                            onChange={(date) => {
+                              const formattedDate = date
+                                ? dayjs(date).format('YYYY-MM-DD')
+                                : '';
+                              handleChange(card.id, 'startDate', formattedDate);
+                            }}
+                          />
+                        </LocalizationProvider>
+                      )}
+                      {m.key === 'endDate' && (
+                        <Box display="flex" alignItems="center">
+                          <LocalizationProvider dateAdapter={AdapterDayjs}>
+                            <DatePicker
+                              label="Till"
+                              disabled={isPresent[card.id]}
+                              minDate={dayjs(card.startDate)}
```

Comment:
TODO: new issue.

With this implementation the user can't input the `endDate` before the `startDate`. It enforces the form to be filled in order, just to be aware.